File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P05LAB_While_Loop/P06_MaxNumber.java
package P05LAB_While_Loop;

import java.util.Scanner;

public class P06_MaxNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        int number = 0, maxNumber = Integer.MIN_VALUE;
        while (!input.equals("Stop")) {
            number=Integer.parseInt(input);
            if (number > maxNumber) {
                maxNumber = number;
            }
            input = scanner.nextLine();
         }
        System.out.println(maxNumber);
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/05.LAB_PB-JS-While-Loop/P06.MaxNumber.js
function maxNumber(input) {
    let biggestNumber = Number.MIN_SAFE_INTEGER;
    let i = 0;
    while (input[i] !== "Stop") {
        let currentNumber = Number(input[i]);
        if (currentNumber > biggestNumber) {
            biggestNumber = currentNumber;
        }
        i++;
    }
    console.log(biggestNumber);
}
maxNumber(["45",
    "-20",
    "7",
    "99",
    "Stop"])

